tests/testthat/test_ExtractData_Soils.R

```diff
@@ -0,0 +1,237 @@
+context("Soil data extraction")
+skip_on_travis();
+skip_on_appveyor();
+skip_on_cran();
+# =============================================================================
+# The tests contained here are still in development and not all inclusive (yet);
+# they do not check for individual value correctness for extraction functions,
+# instead the goal here is to provide tests that are general enough that changing
+# the files from which to extract data from won't cause the tests to fail. Ie. the
+# tests are designed to test the underlining structures created from these functions.
+# =============================================================================
+
+# set stage manually so that future changes won't cause the test to fail ======
+resume <- TRUE
+verbose <- FALSE
+MMC <- environment();
+sim_size <- environment();
+sim_space <- environment();
+fnames_in <- environment();
+SFSW2_glovars <- environment();
+SFSW2_glovars[["slyrs_ids"]] <- seq.int(1, 20, 1);
+SFSW2_glovars[["slyrs_maxN"]] <- 20;
+SWRunInformation <- data.frame(row.names <- c(1,2,3,4,5,6))
+SWRunInformation[,1] <- c(-106.2995, -106.2748, -106.2813, -106.2875, -106.2875, -106.2875)
+SWRunInformation[,2] <- c(35.7655, 35.76451, 35.77990, 35.79530, 35.79530, 35.79530)
+SWRunInformation[,3] <- rep("GriddedFROM100m", 6)
+SWRunInformation[,4] <- rep.int(1, 6)
+colnames(SWRunInformation) <- c("X_WGS84", "Y_WGS84", "SoilTexture_source", "Include_YN_SoilSources")
+sim_size$expN <- 2
+sim_size$runsN_master <- 6
+sim_size$runIDs_sites <- c(1,2,3,5,6)
+sim_size$runsN_sites <- 5
+sim_size$runsN_total <- 12
+sim_size$runIDs_total <- c(1  ,2,  3,  4,  5,  6,  7,  8,  9, 10 ,11, 12)
+sim_size$runsN_job <- 10
+sim_size$runsN_Pid <- 12
+sim_size$runsIDs_todo <- NULL
+sim_size$runsN_todo <- 0
+sim_size$digitsN_total <- 3
+sim_size$runIDs_sites_by_dbW <- c(1,2,3,4,5)
+
+sim_space$scorp <- "point"
+sim_space$crs_sites <- sp::CRS("+init=epsg:4326")
+
+sim_space$sim_res <- NA
+sim_space$run_sites <- sp::SpatialPoints(coords <- SWRunInformation[sim_size[["runIDs_sites"]],
+                                                 c("X_WGS84", "Y_WGS84")],
+                                                 proj4string <- sim_space[["crs_sites"]])
+
+sim_space$sim_crs <- sp::CRS("+init=epsg:4326")
+temp_crs <- as.character(sim_space[["sim_crs"]])
+if (requireNamespace("rgdal", quietly = TRUE)) {
+  temp <- rgdal::checkCRSArgs(temp_crs)
+  stopifnot(temp[[1]])
+  sim_space[["sim_crs"]] <- sp::CRS(temp[[2]])
+}
+
+fnames_in$fslayers <- "/PATH_TO_YOUR_SWRuns_InputData_SoilLayers csv file"
```

Comment:
Move lines where you need to set a path to the top of this file to make more evident for users.